compiler/lib/src/main/scala/util/Error.scala

```diff
@@ -206,6 +209,8 @@ sealed trait Error {
         System.err.println(loc2)
       case SemanticError.PassiveAsync(loc) =>
         Error.print (Some(loc)) ("passive component may not have async input")
+      case SemanticError.PassiveStateMachine(loc) =>
+        Error.print (Some(loc)) ("passive component may not have a state machine")
```

Comment:
I think the error should say "passive component may not have a state machine instance."

Reply:
Fixed